nilearn/plotting/img_plotting.py

```diff
@@ -37,6 +39,50 @@
 # Core, usage-agnostic functions
 
 
+def check_threshold(threshold, data, percentile_calculate, name=None):
```

Comment:
Not sure whether this comment was made before the most recent changes but I don't see too much wrong with this function right now.

I think something which needs to be done is to use it in the plot_connectome code, which hasn't been done yet.